hack/ci/e2e.sh

```diff
@@ -103,6 +103,16 @@ apiVersion: kind.sigs.k8s.io/v1alpha2
 nodes:
 # the control plane node
 - role: control-plane
+  # patch the generated kubeadm config with some extra settings
+  kubeadmConfigPatches:
+  - |
+    apiVersion: kubeadm.k8s.io/v1alpha3
+    kind: ClusterConfiguration
+    metadata:
+      name: config
+    networking:
+      podSubnet: "10.192.0.0/16"
```

Comment:
general kind CI should work without a patch like this.

Reply:
AFAIK podSubnet is required by Calico and Flannel

Reply:
yes, unlike weave, some CNIs need pod network CIDRs.

also e.g. Calico users that want IPv6 need to patch with an IPv6 pod-net CIDR...
it's messy.